pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
@jbw976 I think getting the API routes correct and consistent and doing all those tickets for storage/node labelling are important. Is there a way to clone you and get it all done for 0.4 :)?